src/main/java/eggventory/logic/commands/list/ListStockTypeCommand.java

```diff
@@ -26,20 +27,20 @@ public String execute(StockList list, Ui ui, Storage storage) {
             ui.print(output);
             // Drawing stock data in GUI table.
             ui.drawTable(list.getAllStockTypesStruct());
+        } else if (!list.isExistingStockType(query)) {
+            throw new BadInputException("Invalid command: No such stocktype exists!");
+        } else if (list.isStocktypeZeroQuantity(query)) {
+            output = "There is currently 0 stock with that stocktype.";
```

Comment:
GUI table doesn't update?

Reply:
What is the intended GUI output if it does?

Reply:
an empty table?